### ISSUE TITLE
Generated epub doesn't contain images

### ISSUE THREAD

**Issue by aahmozart** [bartoffw/instabook#2]
Hey there, thanks for availing this extension! It's been a great help in moving my reading from PC to Kobo.

I'm getting this small issue where if I instabook a page the epub won't include the images. A lot of articles contain images so it's very important to have the images within the epub.

You can try this website and it would produce the issue I mentioned:
https://vitalik.eth.limo/general/2024/05/17/decentralization.html

**Comment by bartoffw** [comment]
Thanks for your message. I'll take a look at it this week and see how this can be fixed. The problem may be related to how this specific site is constructed and how it's embedding the images.

**Comment by aahmozart** [comment]
I had the same issue on different sites so I think it's from instabook, but I do run Librewolf which hardens the browser a lot, so there may be something there

**Comment by bartoffw** [comment]
The problem should be fixed now. I've just released version [1.1.2](https://github.com/bartoffw/instabook/releases/tag/1.1.2) to the extension stores so it will be available in your browser soon.

**Comment by bartoffw** [comment]
I also pushed a hotfix as version [1.1.3](https://github.com/bartoffw/instabook/releases/tag/1.1.3) as I found that some images are not visible on some readers because of my recent changes.

**Comment by bartoffw** [comment]
Closing the issue as it's considered fixed.

**Comment by aahmozart** [comment]
Doesn't seem fixed with this article, https://guzey.com/books/why-we-sleep/

I'm on  1.2.8

**Comment by bartoffw** [comment]
Yeah, I can confirm images are missing for this page. I'll take a look at this.

**Comment by bartoffw** [comment]
@aahmozart I've just released version 1.2.9 on all platforms. It should fix the problem of missing images.

**Comment by aahmozart** [comment]
https://www.nationalgeographic.com/travel/article/how-sicily-has-celebrated-its-diversity-through-the-centuries#:~:text=From%20kasbahs%20to%20couscous%E2%80%94how%20Sicily,its%20diversity%20through%20the%20centuries

The epub for this article doesn't show images. I'm on 1.2.10

**Comment by bartoffw** [comment]
I can confirm this is being a problem, I can see the images in the epub file but they are all just links to the online images, not local copies.

**Comment by bartoffw** [comment]
This is fixed in version 1.2.11 that will soon be available on Chrome, MS and Firefox extension stores.
